.github/workflows/build-and-test.yml

```diff
@@ -141,6 +141,28 @@ jobs:
                   name: Wheel for ${{ matrix.os }} ${{ matrix.python-version }} ${{ matrix.python-architecture }}
                   path: dist
 
+            - name: Set up GCP
```

Comment:
Could you explain briefly the GCP / GCS code here and when this will be run? Are artifacts deployed only on push to master?

Reply:
Artifacts will be deployed on any push to any branch.  So on `natcap/invest` the deployment would happen whenever a PR into `master` or `release/*` is merged, or when a release branch is pushed as part of the release process.  If you have the right secret defined in your fork, the deployment would happen on each push there, too.

The reason we have the `if: github.event_name != 'pull_request'` condition is that deploying to GCS requires the service account key to be defined and secrets aren't provided to actions run within PRs (it'd be a pretty glaring security risk if they were!)

Reply:
Thanks!